src/lib/server/xata-old.ts

```diff
@@ -1,7 +1,10 @@
-import { XATA_API_KEY } from '$env/static/private';
+import { XATA_API_KEY, XATA_BRANCH } from '$env/static/private';
 import { XataClient } from './xata';
 
-export const getXataBranch = (branch: string) => {
-	const client = new XataClient({ apiKey: XATA_API_KEY, branch });
-	return client;
+const instances: { [branch: string]: XataClient } = {};
+
+export const getXataClient = (branch: string = XATA_BRANCH || 'main') => {
```

Comment:
would it be safer for the default branch to be 'dev' instead of 'main'?  If XATA_BRANCH is not defined, it probably means there's a bug somewhere right?

Reply:
good point. ideally it never gets undefined it wont be a bug but rather failure of devs to define the variable.

Reply:
this should be default dev